### DISCUSSION TITLE
Can I use this inside a jest test?

### DISCUSSION THREAD

**Discussion by zelief** [ardatan/feTS#1295] [Q&A] [answered]
I get this error when trying to the client for my test:
`ENOENT: no such file or directory, open 'node:stream'` 

![Screenshot from 2024-03-20 17-13-23](https://github.com/ardatan/feTS/assets/5326937/9fac2154-d448-426b-90be-a20536431468)

Can I use this inside Jest environment?

**Comment by ardatan** [comment, marked as answer]
Could you make sure you have the latest version of `@whatwg-node/node-fetch` in your lockfile?
If still doesn't work, could you please create a reproduction on CodeSandbox?
We are already using Jest in our repository so it should work :)

**Reply by zelief** [reply]
Wow, it works!

Thank you so much @ardatan
